src/control/run_go_tests.sh

```diff
@@ -44,6 +44,9 @@ function setup_environment()
 
 	source "${build_source}"
 
+	src_include="$(pwd)/../src/include"
```

Comment:
I would prefer to check for the existence of this directory before adding it as an include path. There are users of this script outside of CI, and if it's called outside of `daos/src/control` then it won't work (or at best will be misleading).

e.g.
```bash
src_include="$(readlink -f $(pwd)/../src/include)"
if [ -d "$src_include" ]; then
	echo "including path \"${src_include}\" in CGO_CFLAGS"
	CGO_CFLAGS+=" -I${src_include}"
fi
```